### PR TITLE
Webpacker restored

### PR DESCRIPTION
**Pull request by hdamico**
## What does this PR do?

Restores Webpacker.

### CODE REVIEW

File: app/javascript/packs/active_admin/print.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/* Active Admin Print Stylesheet */
+@import "~@activeadmin/activeadmin/src/scss/print";
```
Comment (santib) [inline app/javascript/packs/active_admin/print.scss:2]: SCSS should be served by the asset pipeline, not by webpacker.

JS -> webpacker
CSS -> asset pipeline

that's the Rails default

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+// or the `imagePath` JavaScript helper below.
+//
+// const images = require.context('../images', true)
+// const imagePath = (name) => images(name, true)
+
+import 'stylesheets/application'
+
+require('@rails/ujs').start()
+require("turbolinks").start()
+require("@rails/activestorage").start()
```
Comment (santib) [inline app/javascript/packs/application.js:22]: are we using activestorage?

File: config/webpack/environment.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
 const { environment } = require('@rails/webpacker')
+const jquery = require('./plugins/jquery')
```
Comment (santib) [inline config/webpack/environment.js:2]: do we need jquery?

File: package.json [hunk 4]
```diff
@@ -2,14 +2,18 @@
   "name": "github_analyzer",
   "private": true,
   "dependencies": {
+    "@activeadmin/activeadmin": "^2.7.0",
     "@rails/actioncable": "^6.0.0",
     "@rails/activestorage": "^6.0.0",
```
Comment (santib) [inline package.json:7]: I think we don't need these two

File: app/javascript/packs/active_admin/print.scss (hunk 1 above)
Comment (hdamico) [reply app/javascript/packs/active_admin/print.scss:2]: This was generated by the ActiveAdmin, moving to stylesheets.

File: app/javascript/packs/application.js (hunk 2 above)
Comment (hdamico) [reply app/javascript/packs/application.js:22]: Not really, my mistake!

File: config/webpack/environment.js (hunk 3 above)
Comment (hdamico) [reply config/webpack/environment.js:2]: Is being used by Bootstrap.

File: Gemfile [hunk 5]
```diff
@@ -38,7 +34,7 @@ gem 'annotate', '~> 3.0'
 # Use Sidekiq to enqueue job in the background
 gem 'sidekiq', '~> 6.0.2'
 
-gem 'activeadmin', '~> 2.6.0'
+gem 'activeadmin', '~> 2.7'
```
Comment (santib) [inline Gemfile:37]: Looking https://activeadmin.info/0-installation.html section `Upgrading` seems that you might need to update these files:
app/admin/dashboard.rb
config/initializers/active_admin.rb

File: app/javascript/stylesheets/active_admin.scss [hunk 6]
```diff
@@ -8,8 +8,8 @@
 // $sidebar-width: 242px;
 
 // Active Admin's got SASS!
-@import "active_admin/mixins";
-@import "active_admin/base";
+@import "~@activeadmin/activeadmin/src/scss/mixins";
+@import "~@activeadmin/activeadmin/src/scss/base";
```
Comment (santib) [inline app/javascript/stylesheets/active_admin.scss:12]: I know this was autogenerated by active admin and I'm not sure if it's possible what I'm going to ask you but, could you try removing CSS/SCSS from `app/javascript/*` and try to use the asset pipeline for CSS (you might need to restore some active admin CSS files in `app/assets/stylesheets/*`?

Comment (hdamico) [reply app/javascript/stylesheets/active_admin.scss:12]: Yeah, sorry I misunderstood you!

File: Gemfile [hunk 7]
```diff
@@ -7,11 +7,7 @@ ruby '~> 2.6.3'
 gem 'rails', '~> 6.0.2'
 # Use Puma as the app server
 gem 'puma', '~> 4.3'
-# Use SCSS for stylesheets
-gem 'sass-rails', '>= 6'
```
Comment (santib) [inline Gemfile:11]: I don't think you need to remove this. We are going to have CSS via the asset pipeline so we might want to have SASS.

If you create a new rails 6 app you'll see this gem is present along with webpacker gem

File: config/initializers/active_admin.rb [hunk 8]
```diff
@@ -4,7 +4,7 @@
   # Set the title that is displayed on the main layout
   # for each of the active admin pages.
   #
-  config.site_title = 'Github Analyzer'
+  config.site_title = Rails.application.class.name.split('::').first.titlecase
```
Comment (horacio) [inline config/initializers/active_admin.rb:7]: Nit:

```suggestion
  config.site_title = Rails.application.class.name.deconstantize
```

**Pull request merged**